pipeline/scripts/5/1/tier-0/test-rhceph-deployment-features.sh

```diff
@@ -5,7 +5,7 @@ random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
 instance_name="ci-${random_string}"
 platform="rhel-8"
 rhbuild="5.1"
-test_suite="suites/pacific/cephadm/tier_0_cephadm.yaml"
+test_suite="suites/pacific/cephadm/tier0_5_1_cephadm.yaml"
```

Comment:
Can we keep the file naming consistent? For example: `tier_0_cephadm_5.1.yaml`

Reply:
As per our discussion, separated out build-version values against test suite name.